code-transform/src/common.ts

```diff
@@ -4,15 +4,11 @@ export function getFunctionName(e: BabelTypes.FunctionDeclaration, t: typeof Bab
   return t.isIdentifier(e.id) ? e.id.name : undefined;
 }
 
-interface WithBabelLeadingComments {
-  leadingComments: ReadonlyArray<BabelTypes.Comment> | null;
-}
-
-export function isExposedStatement(node: WithBabelLeadingComments) {
+export function isExposedStatement(node: BabelTypes.Statement) {
   return node.leadingComments && node.leadingComments.some((comment) => /@expose/.test(comment.value));
 }
 
-export function isTypeScriptGeneratedExport(e: object, t: typeof BabelTypes, funcName: string) {
+export function isTypeScriptGeneratedExport(e: BabelTypes.Statement, t: typeof BabelTypes, funcName: string) {
```

Comment:
Actually used in `index.ts` as `sibling.node` where it's a `BabelTypes.Node`. (TBH That's a very fragile typescript output check 😞 )